readers.py

```diff
@@ -156,7 +157,14 @@ def lis_cube(lis_dir, lis_input_file, var, start, end, subfolder = "SURFACEMODEL
     :param int h: UTC time at which LIS outputs (e.g., daily outputs at 12 UTC: h = 12)
     :param str d: domain (in filename)
     :param str freq: temporal resolution of the output
+    :param bool date_shift: use this option to shift the LIS output date with "freq". Recommended option for "_tavg" output.
     """
+
+    # warnings related to the date_shift option
+    if ("_tavg" in var) and (not date_shift):
+        print("It is recommended to set date_shift=True for averaged outputs.")
```

Comment:
Just a thought: it depends on the time zone right? We usually output the SURFACEMODEL at UTC0, meaning that for Europe it corresponds +- to a day shift but that's not necessarily the case for other regions. Would you suggest to adapt the outputing of the SURFACEMODEL according to the time zone?

Reply:
My understanding was that the time averaged output was the average since the previous output. So for daily output, the last 24h; for hourly output, the last hour. Is this not correct? And if it is, could you elaborate how the time zone would affect this?